### PR TITLE
Updated biography

### PR DESCRIPTION
**Pull request by GiuliaMantuano**
I'll let @mashooq decide whether to merge it or not :)
To the native speakers, please let me know if you notice any mistakes. Thank you!

### CODE REVIEW

File: _data/team.yml [hunk 1]
```diff
@@ -157,7 +157,12 @@
...
+  fullDescription: >
+    <p>Giulia has worked as a design consultant and art director in an integrated communications agency for more than ten years.</p>
+    <p>During her career she has developed both the design talent and business knowledge necessary for creating meaningful holistic brand experiences.</p>
+    <p>Giulia joined Codurance as an Apprentice driven by the desire to apply the Software Craftsmanship mindset and values to the whole product development process.</p>
+    <p>She is a Certified Scrum Master and Certified Scrum Product Owner with a strong focus on the present and the future of a product, ensured by her Integrated Design background and the mastering of User Experience Design skills.</p>
+    <p>Giulia bridges stakeholders and development teams: she facilitates their collaboration, aiming at designing well balanced solutions with business needs, technical feasibility and user expectations.</p>
```
Comment (bnathyuw) [inline _data/team.yml]: How about: 'she facilitates their collaboration in designing solutions that balance business needs, technical feasibility and user expectations.'

File: _data/team.yml [hunk 2]
```diff
@@ -157,7 +157,12 @@
...
+  fullDescription: >
+    <p>Giulia has worked as a design consultant and art director in an integrated communications agency for more than ten years.</p>
+    <p>During her career she has developed both the design talent and business knowledge necessary for creating meaningful holistic brand experiences.</p>
+    <p>Giulia joined Codurance as an Apprentice driven by the desire to apply the Software Craftsmanship mindset and values to the whole product development process.</p>
+    <p>She is a Certified Scrum Master and Certified Scrum Product Owner with a strong focus on the present and the future of a product, ensured by her Integrated Design background and the mastering of User Experience Design skills.</p>
```
Comment (bnathyuw) [inline _data/team.yml]: I would suggest 'backed up by her Integrated Design background and the masteryn [or 'extensive knowledge'] of User Experience Design skills.'

**Review by bnathyuw — COMMENTED** [review]
Looks good. I've made a couple of small suggestions.

File: _data/team.yml (hunk 1 above)
Comment (GiuliaMantuano) [reply _data/team.yml]: Beautiful!

**Pull request merged**